scout/adapter/mongo/query.py

```diff
@@ -163,7 +358,7 @@ def build_query(self, case_id, query=None, variant_ids=None, category='snv'):
                         ]
                     }
                 )
-            logger.debug("Adding gnomad_frequency to query")
+            LOG.debug("Adding gnomad_frequency to query")
```

Comment:
This function gets fairly long as well. Would splitting into a few broader categories according to type; e.g. frequency, function, sv specific, cancer specific make it better?

Reply:
What about looping through the list of secondary criteria and grouping them by filter construction? I have given it a try and the function is a bit shorter now. See if you like it, otherwise I can change!